package.json

```diff
@@ -18,19 +18,19 @@
   },
   "main": "./lib/coffee-script/coffee-script",
   "bin": {
-    "coffee": "./bin/coffee",
-    "cake": "./bin/cake"
+    "iced": "./bin/coffee",
```

Comment:
What do you think about an install as `iced3` so that we can run both side-by-side for a while as we test?